src/Cardano/Prelude/GHC/Heap/NormalForm/Classy.hs

```diff
@@ -28,27 +28,27 @@ module Cardano.Prelude.GHC.Heap.NormalForm.Classy (
   , AllowThunk(..)
   ) where
 
-import Cardano.Prelude.Base
-
-import Data.Foldable (toList)
-import Data.Sequence (Seq)
-import Data.List.NonEmpty (NonEmpty)
-import Data.Time
-import GHC.Exts.Heap
-import Prelude (String)
-
-import qualified Data.ByteString as BS.Strict
-import qualified Data.ByteString.Lazy as BS.Lazy
-import qualified Data.IntMap as IntMap
-import qualified Data.Map as Map
-import qualified Data.Set as Set
-import qualified Data.Text as Text.Strict
-import qualified Data.Text.Lazy as Text.Lazy
-import qualified Data.Vector as Vector.Boxed
-import qualified Data.Vector.Unboxed as Vector.Unboxed
-
-import Cardano.Prelude.GHC.Heap.Tree
+import           Cardano.Prelude.Base
+
+import           Data.Foldable                       (toList)
+import           Data.List.NonEmpty                  (NonEmpty)
+import           Data.Sequence                       (Seq)
+import           Data.Time
+import           GHC.Exts.Heap
+import           Prelude                             (String)
+
+import qualified Data.ByteString                     as BS.Strict
+import qualified Data.ByteString.Lazy                as BS.Lazy
+import qualified Data.IntMap                         as IntMap
+import qualified Data.Map                            as Map
+import qualified Data.Set                            as Set
+import qualified Data.Text                           as Text.Strict
+import qualified Data.Text.Lazy                      as Text.Lazy
+import qualified Data.Vector                         as Vector.Boxed
+import qualified Data.Vector.Unboxed                 as Vector.Unboxed
```

Comment:
Should probably revert the formatting here :smile:

Reply:
Agreed.

Reply:
Oh, you did already. Good man :)